.NET/Microsoft.Recognizers.Text.DataTypes.DataDrivenTests/TestTimex.cs

```diff
@@ -127,6 +127,13 @@ public void DataTypes_Timex_FromTime()
             Assert.AreEqual("T23:59:30", TimexProperty.FromTime(new Time(23, 59, 30)).TimexValue);
         }
 
+        [TestMethod]
+        public void DataTypes_Timex_FromDateTimeRange_ToString()
+        {
+            var timex = new TimexProperty("(2022-03-15T16,2022-03-15T18,PT2H)");
```

Comment:
This is not technicaly correct, as the timex specified doesn't correspond to just one date, but a date range. We need to properly correct this in a more general way.

Reply:
Technically what I did here is make it not throw the exception, without changing the original logic of the output - this has always returned incorrect results for a date range, the logic was always wrong for that, but the exception is the issue I'm trying to fix here, because it's very hard to work around it without reimplementing a bunch of stuff on the user side.

I would very much like to have this return the correct output for a date range, that was going to be my next issue to open. Can we fix this exception now and I'll put together a separate PR to fix the output?

Reply:
In this case, please add a code comment in that unit test along the lines of:
`\\ @TODO Test documenting workaround to avoid exceptions when getting a timex. Proper timex generation fix needed.`

Thanks!

Reply:
Added the TODO and resolved conflicts 👍